build.properties

```diff
@@ -83,3 +84,13 @@ jfx.build.jdk.buildnum.min=28
 # that is supported. It must be <= jfx.gradle.version.
 jfx.gradle.version=5.3
 jfx.gradle.version.min=4.8
+
+# Toolchains
+jfx.build.linux.gcc.version.min=gcc7.3.0-OEL6.4+1.1
+jfx.build.windows.msvc.version.min=VS2017-15.5.5+1.1
+jfx.build.macosx.xcode.version.min=Xcode9.4-MacOSX10.13+1.0
```

Comment:
I think for the above three, you can just drop the `.min` (although it would be helpful to note what the minimum is in a comment, we probably don't need a variable for the min unless we are going to check and fail the build if it is below the minimum).

Reply:
Thanks Kevin. I will drop the min suffix.